src/tools/miri/tests/pass/drop_in_place_null.rs

```diff
@@ -0,0 +1,7 @@
+// Make sure that dropping types with no drop glue is DB even for invalid pointers.
```

Comment:
This is totally against what we [document](https://doc.rust-lang.org/nightly/core/ptr/fn.drop_in_place.html#safety). So I think we should make this UB. If that's a problem, clearly the documentation must be updated.